### ISSUE TITLE
Can't see reactions to images in chat (except your own)

### ISSUE THREAD

**Issue by qoqobolo** [status-im/status-react#14920]
# Bug Report

### Reproduction

1. UserA: send a picture in a channel
2. UserB: set a reaction to the picture
3. UserA: check the reaction 

Works fine for other types of messages.

#### Expected behavior
Can see reaction

#### Actual behavior
No reaction

### Additional Information

- Status version: nightly 26/01
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

**Comment by rasom** [comment]
@OmarBasem 
so the reason for this is that we do not send `album-id` to recipient and thus both sides use different `message-id` for reactions (sender uses `album-id`, recipient uses `message-id`). Of course that also impacts how images are shown on both sides because on recipient side they are not grouped into albums and are shown as separate messages. Is there some ongoing work to send `album-id` along with the image?

**Comment by OmarBasem** [comment]
@rasom Yes, there is a PR https://github.com/status-im/status-go/pull/3121 to deliver the image fields (album-id, width, height) to the receiver.

**Comment by rasom** [comment]
@OmarBasem @qoqobolo then there is no point in fixing this issue separately, it should be solved by https://github.com/status-im/status-go/pull/3121

**Comment by qoqobolo** [comment]
@rasom okay, thanks for looking at this anyway.

@OmarBasem we could test the fixes for this one and for https://github.com/status-im/status-mobile/issues/14919 once https://github.com/status-im/status-go/pull/3121 is ready. Please ping us in a corresponding PR when you create one in the mobile repo.
Will keep these issues open for now.
Thanks

**Comment by OmarBasem** [comment]
Hi @qoqobolo,
The corresponding PR has been merged into status-go. I see status-mobile is already using the latest status-go commit, so there is no need for a PR. The square images issue https://github.com/status-im/status-mobile/issues/14919 should be fixed now on latest develop. Also if this reactions issue was due to the album-id, then it may be fixed too.

**Comment by qoqobolo** [comment]
Hey @OmarBasem , thanks for the update.
Unfortunately, I am unable to check the fix for reactions due to this issue: https://github.com/status-im/status-mobile/issues/14979
May it be related to `go#3121`? The issue has returned since yesterday's nightly, but I'm not sure what time exactly the PR was merged. Can you take a look, please?

**Comment by OmarBasem** [comment]
@qoqobolo so as you are saying long tap is not working properly, I will fix it. But long-pressing on the image from the edge does open the menu, and the reactions seem to not be delivered. So the issue is still there.

**Comment by OmarBasem** [comment]
@qoqobolo actually it is fixed. I was running an older version on my device. I will fix the menu issue, and you will be able to check it.

**Comment by qoqobolo** [comment]
Thanks @OmarBasem!

**Comment by OmarBasem** [comment]
Closing this issue as https://github.com/status-im/status-mobile/pull/14996 is now merged.
